components/activity_log_modal/activity_log_modal.tsx

```diff
@@ -82,29 +89,24 @@ export default class ActivityLogModal extends React.PureComponent {
     }
 
     render() {
-        let content;
-        if (this.props.sessions.loading) {
-            content = <LoadingScreen/>;
```

Comment:
I don't think we should be removing this logic here. I think it's causing some of the tests to fail by removing this.

Reply:
in the index we bind sessions props to this method:
 
 `sessions: getUserSessions(state),`
 
 which returns an array of type `Sessions`.
 
I don't think the code would reach the `if` block. let me know if I'm wrong. @devinbinnie

Reply:
I took a look at the reducer and it looks like it always returns an array, but I may be missing something. I'll defer to @devinbinnie here.

Reply:
It does look that way, weird that this code was never removed. If we're removed this code, can we also remove the tests that are looking for this code? Thanks!